### DISCUSSION TITLE
V1.5.1 - The Support Update

### DISCUSSION THREAD

**Discussion by Joalor64** [Joalor64/Joalor64-Engine#27] [Announcements]
This new update adds a lot of support for a lot of things!

Sorry this took so long. It was just because school was bothering me.

But hey, it's here now!

So with all of that out of the way, enjoy!

<hr /><em>This discussion was created from the release <a href='https://github.com/Joalor64GH/Joalor64-Engine/releases/tag/v1.5.1'>V1.5.1 - The Support Update</a>.</em>

**Comment by ghost** [comment]
Joalor please help me

**Comment by ghost** [comment]
I can't add hscript in my engine!

**Comment by ghost** [comment]
help please🙏🙏🙏🙏😩😩😩

**Comment by Joalor64** [comment]
okay sure 👍

**Comment by ghost** [comment]
help me real?

**Comment by Joalor64** [comment]
yes

**Reply by ghost** [reply]
> yes

give discord

**Comment by ghost** [comment]
give your discord

**Comment by ghost** [comment]
if have

**Comment by Joalor64** [comment]
P00P36#7620

**Reply by ghost** [reply]
my name in discord "Pico Soft''

**Reply by Joalor64** [reply]
okay

**Reply by ghost** [reply]
open your discord

**Reply by ghost** [reply]
![Screenshot_20221128-165801](https://user-images.githubusercontent.com/111678905/204296220-8444f345-3c19-4b76-9ccd-e7641b42de68.jpg)

**Comment by ghost** [comment]
help please

**Comment by ghost** [comment]
Joalor

**Comment by ghost** [comment]
![Screenshot_20221128-203726](https://user-images.githubusercontent.com/111678905/204346245-4f92a290-e9eb-4ec1-89f4-a96578535ece.jpg)
![Screenshot_20221128-203351](https://user-images.githubusercontent.com/111678905/204346260-b5693903-34b8-450a-b2e8-0969e3e03f61.jpg)

**Comment by ghost** [comment]
file name bug

**Comment by ghost** [comment]
error begin

**Comment by ghost** [comment]
bug not fix

**Comment by ghost** [comment]
you it can fix?

**Reply by ghost** [reply]
joalor fix it error

**Comment by ghost** [comment]
idk how fix it

**Comment by ghost** [comment]
joalor help please fix it error

**Comment by ghost** [comment]
you forgot install hscript

**Comment by Joalor64** [comment]
oh

**Comment by ghost** [comment]
please fix it

**Comment by Joalor64** [comment]
okay
